src/H5Pdxpl.c

```diff
@@ -2610,6 +2621,39 @@ H5Pget_no_selection_io_cause(hid_t plist_id, uint32_t *no_selection_io_cause /*o
     FUNC_LEAVE_API(ret_value)
 } /* end H5Pget_no_selection_io_cause() */
 
+/*-------------------------------------------------------------------------
+ * Function:	H5Pget_actual_selection_io_mode
+ *
+ * Purpose:	    Retrieves actual selection I/O mode
+ *
+ * Return:	    Non-negative on success/Negative on failure
+ *
+ * Programmer:	Vailin Choi
+ *              April 27, 2023
+ *-------------------------------------------------------------------------
+ */
+herr_t
+H5Pget_actual_selection_io_mode(hid_t plist_id, uint32_t *actual_selection_io_mode /*out*/)
+{
+    H5P_genplist_t *plist;
+    herr_t          ret_value = SUCCEED; /* return value */
+
+    FUNC_ENTER_API(FAIL)
+    H5TRACE2("e", "ix", plist_id, actual_selection_io_mode);
+
+    /* Get the plist structure */
+    if (NULL == (plist = H5P_object_verify(plist_id, H5P_DATASET_XFER)))
+        HGOTO_ERROR(H5E_ID, H5E_BADID, FAIL, "can't find object for ID")
```

Comment:
Should probably be H5E_PLIST, not H5E_ID

Reply:
This is consistent with how H5P_object_verify is handled everywhere else. I don't have strong feelings either way but I think if we're going to change it we should change it everywhere in a separate PR. Or we can just wait until we're able to get rid of major/minor errors classes.